### DISCUSSION TITLE
Push a new picture doesn't replace the old one.

### DISCUSSION THREAD

**Discussion by madyanes** [community/community#48795] [Other Feature Feedback, Questions, & Ideas] [answered]
I have pushed a new picture with the same name, but when I opened it, the old picture is opened instead of the new picture.

Edit: sorry, I think that was because my internet connection. Issue closed.

**Comment by sbhask** [comment, marked as answer]
Could this be an issue with your network?

**Reply by madyanes** [reply]
Yes, it was my network problem. Sorry.
